tests/requirements.txt

```
@@ -4,7 +4,7 @@ ruff==0.0.287
 
 # Runtime requirements
 Django>=4.2,<5.1
-wagtail>=4.1
+wagtail>=6.2,<6.3
```

```suggestion
wagtail>=5.2,<6.3
```

allthough I would even go as far as `<6.5`